common/authorization/authorizer.go

```diff
@@ -115,6 +117,27 @@ type FilteredRequestBody interface {
 	SerializeForLogging() (string, error)
 }
 
+type simpleRequestLogWrapper struct {
+	request interface{}
+}
+
+func (f *simpleRequestLogWrapper) SerializeForLogging() (string, error) {
+	if f.request == nil || reflect.ValueOf(f.request).IsNil() {
```

Comment:
This is a bit inefficient, but this is a minor thing in comparison to the next json.Marshal, so should be fine to use

Reply:
Agreed, I added a comment outlining this